libparsec/crates/serialization_format/src/protocol.rs

```diff
@@ -4,6 +4,9 @@
 #[path = "./protocol_python_bindings.rs"]
 pub(crate) mod python_bindings;
 
+#[path = "./protocol_tests.rs"]
+pub(crate) mod tests;
```

Comment:
```suggestion
pub(crate) mod cmds_tests_generator;
```
calling it `test` is misleading given we may think it is tests for the current crate (and not a regular macro providing used by another crate to generate tests !)